code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -143,6 +146,9 @@
 
 	set_sprite(feature_list[deconstruct_block(get_uni_feature_block(features, dna_block), feature_list.len)])
 
+///Colorizes the limb it's inserted to, if required.
+/obj/item/organ/external/proc/override_color(rgb_value)
+	return
```

Comment:
TBH with you I'm not sure you really need to have a var to prevent this proc call, but it's not a huge deal either way.

Reply:
Less proc overhead.